swagger-spec/swagger.yaml

```diff
@@ -709,9 +705,92 @@ tags:
+      the request and get links to other resources.
+
+  - name: Registration Schemas
+    description: >-
+      A Registration Schemas defines the range of valid responses to a registration. Each Registration
+      Schema is composed of <a href="#tag/Registration-Schema-Blocks">"blocks"</a> that define the individual questions a user responds to. Registration Schemas are
+      created by the Center for Open Science or an affiliated institution and periodically migrated into the system.
+      Registration Schemas are updated and deactivated using an internal versioning system.
+
+  - name: Registration Schema Blocks
+    description: >-
+      Registration Schema Blocks are read-only entities, they represent question prompts and form inputs known as
+      "blocks" that make up a Registration Schemas. Each block has a designated `block_type` that determines what type
+      of information that Schema Block's corresponding block in the Schema Response.
+
+  - name: Schema Responses
+    description: >-
+      Schema Responses contain user supplied responses to a Registration Schema. A Schema Response
+      aggregates all of the the responses for a given version of a Registration's Registration Schema.
+      Schema Responses are created when a Registration is completed. Once a Registration is completed the
+      Schema Responses for a Registration can be revisied and updated by using it's associated action endpoints.
+
+
+      There are a few states for Schema Responses:
+
+        - `initial` is the state of a Schema Response on a newly registered Registration, to edit a Schema Response you
+        must create a Schema Response Action that triggers a new submission.
+
+        - `in_progess` is the state of a Schema Response where the response is editable and still private, Schema
+        Responses are editted via a PATCH request as specified below.
+
+        - `unapproved` is the state of a Schema Response where edits have been made and "locked-in", now contributors
+        have the ability to reject the changes, however if they are not rejected changes are automatically approved
+        after 48 hours. If `unapproved` Schema Responses are rejected, they are returned to the `in_progress` state. If
+        `unapproved` Schema Responses are approved they either enter an `approved` state or go into a
+        `pending_moderation` to be accepted or denied by a moderator.
+
+        - `pending_moderation` is the state of a Schema Response where moderators have an opportunity to reject or
+        approve a Schema Response that has been approved by it's contributors. This state is only reachable for Schema
+        Responses that are associated with a Registration that has Registration Provider a moderated workflow.
+
+        - `approved` is the state of a Schema Response where it is public and immutable, in order to update an approved
```

Comment:
"in order to update an approved..." This makes it seem like `approved` responses can be changed. I think "If further edits are needed, a new Schema Response must be created" or something of that wording is better